### ISSUE TITLE
M3AE pre-trained model

### ISSUE THREAD

**Issue by nooralahzadeh** [baeseongsu/ehrxqa#7]
Hi,
Could you share the pre-trained VLP models such as M3AE?

Thanks

**Comment by baeseongsu** [comment]
Hi @nooralahzadeh,

I've uploaded two pre-trained M3AE models used in our paper:
- `m3ae_medicat_roco.ckpt`: Reproduced version of the original M3AE model.
- `m3ae_mimiccxr.ckpt`: Pre-trained on a subset of MIMIC-CXR-JPG images. (This file will be uploaded soon, within 30 mins.)

You can find them in this [link (via Dropbox)](https://www.dropbox.com/scl/fo/3gym2a4xgf4fru0v2xqku/APa3WRX8nwFuvrwCUfdYLiY?rlkey=xgzu1mzca2dthx9940kmt7pbh&st=384uc8za&dl=0).

Best,
Seongsu

**Comment by nooralahzadeh** [comment]
Hi @baeseongsu 
Thanks for the prompt answer and also for providing the checkpoints. 
It would be great if you could help me to find the grounTruth label set for the VQA task. 

Thanks,
Farhad

**Comment by baeseongsu** [comment]
Hi, @nooralahzadeh 

You can download full VQA datasets with ground truth labels from [here](https://physionet.org/content/mimic-ext-mimic-cxr-vqa/1.0.0/).

Best,
Seongsu

**Comment by nooralahzadeh** [comment]
Thanks.

**Comment by nooralahzadeh** [comment]
Hello,
Thanks for providing the M3AE checkpoints. Could you please also provide the values that you have used to test the `m3ae_mimiccxr.ckp `. Did you use this one as below: 

`python main.py with data_root=data/finetune_arrows/ \
 num_gpus=${num_gpus} num_nodes=1 \
 task_finetune_irtr_roco get_recall_metric=True \
 per_gpu_batchsize=${per_gpu_batchsize} \
 clip16 text_roberta \
 image_size=288 \
 test_only=True \
 tokenizer=downloaded/roberta-base \
 load_path=m3ae_mimiccxr.ckpt`

**Comment by baeseongsu** [comment]
Hi @nooralahzadeh,

Yes, I've used a similar code script structure. However, the checkpoints I gave you are for the pre-trained VLP model, not fine-tuned on the MIMIC-CXR-VQA dataset. So I will upload several checkpoints which are fine-tuned models so that you can infer directly. You can check our experiment code for the M3AE model [here](https://github.com/baeseongsu/mimic-cxr-vqa/tree/develop-exp/mimiccxrvqa/exp/M3AE).

**Comment by nooralahzadeh** [comment]
I would be grateful if you could upload the VQA checkpoints, we would like to reproduce your results.

**Comment by baeseongsu** [comment]
Hi @nooralahzadeh,

You can find them in this [link (via Dropbox)](https://www.dropbox.com/scl/fo/t93ptf4q7klq36gvblrro/APAFf2udsN3W8OPNxKXasvs?rlkey=lhur50wgjkh1u3cnqck4e51xf&st=5pkjsbxc&dl=0).

If you have any problems with the experiments, please feel free to reach out.

Best,
Seongsu

**Comment by nooralahzadeh** [comment]
Thanks,
